src/components/custom-field-input-date/custom-field-input-date.jsx

```diff
@@ -59,6 +59,14 @@ const CustomFieldInputDate = forwardRef(function CustomFieldInputDate(props, ref
     setShouldFocusInput(false);
   }, [shouldFocusInput]);
 
+  useEffect(() => {
+    if (ref && currentValue) {
```

Comment:
I have a sneaky suspicion that we still want to call `props.onChange` when `currentValue` is undefined or an empty string.

Reply:
So the problem with undefined, is that it this will get called on first render, and that was what we were attempting to guard against here ... Empty string should def pass through though.